site/docs/pages/swap/types.mdx

```diff
@@ -180,6 +180,16 @@ type SwapReact = {
 };
 ```
 
+## `SwapSettingsReact`
+
+```ts
+type SwapSettingsReact = {
+  className?: string; // Optional className override for top div element.
+  icon?: ReactNode; // Optional icon override
+  text?: string; // Optional text override
+};
```

Comment:
We probably need to document other types, right?

Reply:
Yes, should I document them in a follow up PR? I'll remove SwapSettingsReact for now and add all docs/ changes at once with the examples

Reply:
removed. Will add all types in during the "documentation" PR